### ISSUE TITLE
-dx11 still standard?

### ISSUE THREAD

**Issue by bluenote73** [Healix/Gw2Launcher#179]
Sorry to put this question here, I wasn't sure of a better place. I use -dx11 in my launch options, but now that dx9 is being deprecated in the client, has this behaviour changed? Should I be doing anything different? thx

**Comment by Healix** [comment]
You don't need to do anything. -dx11 will force the use of DX11. They did make DX11 enabled by default, but that's only for new Local.dat files.

**Comment by bluenote73** [comment]
Thank you! Is there a way I can confirm in game that I am in fact on dx11?

**Comment by Healix** [comment]
No, the game itself won't tell you. If you want to confirm it, right click any account in Gw2Launcher > Tools > Guild Wars 2 > Verify usage of DX11.

**Comment by bluenote73** [comment]
thank you!!
